packages/react/package.json

```diff
@@ -67,7 +67,7 @@
     "lodash": "4.17.21",
     "mapbox-gl": "1.13.1",
     "maplibre-gl": "2.1.9",
-    "maplibre-gl-js-amplify": "2.0.2",
+    "maplibre-gl-js-amplify": "^2.0.4",
```

Comment:
Is there a reason for not pinning the dependency? We do that for other direct dependencies

Reply:
Good catch, that was a mistake!